workflow/cron/operator.go

```diff
@@ -300,6 +300,7 @@ func (woc *cronWfOperationCtx) enforceHistoryLimit(ctx context.Context, workflow
 
 	var successfulWorkflows []v1alpha1.Workflow
 	var failedWorkflows []v1alpha1.Workflow
+	var errCount int
```

Comment:
A couple of options here:
1. Track the total count of failed workflows
1. Track the count of each unique errors